tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -714,39 +715,7 @@ describe(__filename, () => {
         slug: collection.slug,
         editing: true,
         errorHandlerId: errorHandler.id,
-        page,
-        username: signedInUsername,
-      }),
-    );
-  });
-
-  it('dispatches addAddonToCollection with a default page of 1 when selecting an add-on', () => {
```

Comment:
This test case was removed because this component used to contain logic that defaulted the `page` to `1` if no page was specified, but now this component always receives a `filters` prop instead and therefore has no logic for specifying a default value for `page`.